### DISCUSSION TITLE
Is this a hard or soft fork of firefox?

### DISCUSSION THREAD

**Discussion by michaelCTS** [Floorp-Projects/Floorp#669] [Q&A] [answered]
Waterfox and other forks are soft forks of firefox that "just" add patches on top of firefox, but a hard fork would start doing it's own stuff and not following firefox development.

What kind of a fork is this project?

**Comment by CutterKnife** [comment, marked as answer]
Floorp is based on Firefox ESR, with many original features implemented natively.
By your standards, this is a soft fork.

**Reply by michaelCTS** [reply]
Thank you!
